web/profiles/custom/os2loop/themes/os2loop_theme/assets/search.scss

```diff
@@ -7,34 +30,136 @@
     display: none;
   }
 
-  .input-group-icon {
-    color: $grey;
-    background: none;
-    border-right-color: transparent;
+  .form-autocomplete {
+    border-radius: 1em;
+
+    // transition: border-radius .1s;
+
+    // Make room for background images
+    padding-left: 50px;
+    padding-right: 50px;
 
-    i {
-      // Squash the search icon.
-      line-height: 0;
+    background: {
+      image: $search-icon;
+      position: $search-icon-padding center;
+      size: 24px 24px;
+      repeat: no-repeat;
     }
   }
 
-  .form-control {
-    border-left-color: transparent;
+  .search-api-autocomplete-has-suggestions {
+    .form-autocomplete {
+      border-bottom-left-radius: 0;
+      border-bottom-right-radius: 0;
+    }
+  }
+}
+
+.search-api-autocomplete-search.ui-widget.ui-widget-content {
+  padding-bottom: 1em;
+  border-bottom-left-radius: 1em;
+  border-bottom-right-radius: 1em;
+  border-top: none;
+}
+
+// @see search_api_autocomplete/css/search_api_autocomplete.css
```

Comment:
why this?

Reply:
Sloppy programming and missing clean up. Now the comment says what it's supposed to say.